src/store/messagesStore.js

```diff
@@ -823,7 +823,7 @@ const actions = {
 	 */
 	async clearLastReadMessage(context, { token, updateVisually = false }) {
 		const conversation = context.getters.conversations[token]
-		if (!conversation || !conversation.lastMessage) {
+		if (!conversation/* FIXME || !conversation.lastMessage */) {
```

Comment:
Why was it commented out ? 
It is still needed to not to cause an error of reading undefined. `conversation.lastMessage.id`